components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.hpp

```diff
@@ -112,6 +112,16 @@ namespace scream {
         void compute_cloud_area(
                 int ncol, int nlay, int ngpt, Real pmin, Real pmax,
                 const real2d& pmid, const real3d& cld_tau_gpt, real1d& cld_area);
+        /*
```

Comment:
This is probably fine for now, but I think this might be where my code is a bad example to follow. It looks like your diagnostics don’t use anything inherently tied to RRTMGP, and thus to YAKL, so it would probably be preferable to implement this in Kokkos and keep separate from the `scream_rrtmgp_interface` code. I think the general guidance is “implement everything we can in Kokkos, except when we can’t or it doesn’t make sense to”. Our fork of RRTMGP was written in YAKL, so functions which need to work directly with RRTMGP are stuck in here to share data types (we have to do deep copies between YAKL arrays and Kokkos views generally due to differences in layout I think).